echopype/clean/api.py

```diff
@@ -38,16 +43,24 @@ def mask_transient_noise(
     ----------
     ds_Sv : xarray.Dataset
         Calibrated Sv data with depth data variable.
-    func: str, default `nanmean`
+    func: str, default "nanmean"
         Pooling function used in the pooled Sv aggregation.
```

Comment:
I feel we should default to "nanmedian" since that is what the original method said.

Reply:
Made this change